takwimu/models/dashboard.py

```diff
@@ -217,6 +217,19 @@ def value_for_form(self, value):
 ]
 
 
+class FlourishWidget(blocks.StructBlock):
+    label = blocks.CharBlock(required=False, help_text="This widget's tab label on the indicator")
+    title = blocks.CharBlock(required=False)
```

Comment:
Lets require titles because when it will be needed during downloads for example. The downloaded file needs a good name.

Reply:
I've just moved the block definition from inline to class without any changes. The original block:

```
    flourish = blocks.StructBlock(
            ('label', blocks.CharBlock(required=False,
                                       help_text="This widget's tab label on the indicator")),
            ('title', blocks.CharBlock(required=False)),
            ('hide_title', blocks.BooleanBlock(default=False, required=False)),
            ('html', DocumentChooserBlock(required=True)),
            ('sdg', blocks.ChoiceBlock(required=False, choices=sdg_choices,
                                       label='SDG Goal')),
            ('source', blocks.RichTextBlock(
                features=['link'], required=False)),
        ],
        icon='code',
        template='takwimu/_includes/dataview/code.html'
    )
```

Hoping introducing the constraint now won't break anything.